### DISCUSSION TITLE
Looking for tool that allows me to load & test onnx file

### DISCUSSION THREAD

**Discussion by Desperado17** [onnx/onnx#6285] [Q&A]
Greetings,

I'm looking for a tool that allows me to visualize an onnx neural network and show the content of the nodes for an example input (the network is an object detection network takes images as input and produces bounding boxes as output). I want to examine the workings of the network for private research.

I tried qnnx modifier but it only seems to visualize the network itself it doesn't allow putting in an input image and examining the intermediate stages, right?

Regards

**Comment by andife** [comment]
Maybe https://github.com/zetane/zetaforge ?
